src/core_plugins/kibana/server/tutorials/netflow/index.js

```diff
@@ -8,11 +8,11 @@ export function netflowSpecProvider() {
     id: 'netflow',
     name: 'Netflow',
     category: TUTORIAL_CATEGORY.SECURITY,
-    shortDescription: 'Collect Netflow records sent by a Netflow exporter',
-    longDescription: 'The Logstash Netflow module simplifies the collection, normalization, and visualization of network flow data. ' +
-      'With a single command, the module parses network flow data, indexes the events into Elasticsearch, and installs a suite of Kibana ' +
-      'dashboards to get you exploring your data immediately. Logstash modules support Netflow Version 5 and 9. [Learn more]' +
-      '({config.docs.logstash}/netflow-module.html) about the Netflow module',
+    shortDescription: 'Collect Netflow records sent by a Netflow exporter.',
+    longDescription: 'The Logstash Netflow module parses network flow data, ' +
```

Comment:
I imagine users who end up on this page are familiar with Netflow concepts and terminlogy. In Netflow parlance a "collector" is a well-defined concept so I'd like that to be reflected here. So could we make this "collects and parses" instead of just "parses"?

Reply:
It's good to use industry-standard terminology.

Reply:
For some reason this PR was merged without addressing this comment. I've pushed https://github.com/elastic/kibana/commit/629f7d8c48c0ed811df5b886d79e85919cb1354b directly to `master` with this change now.